misc/gui_linux.sh

```diff
@@ -0,0 +1,157 @@
+	;;
+
+"ServiceUpNot")
+	data=$(zenity --forms --text="Pass if Service is not running" --add-entry "Message" --add-entry="Points" --add-entry="Service" --separator=$'\034')
+	IFS=$'\034' read -r mess pts arg1 <<< "$data"
+	vulns="${vulns}\n[[check]]\nmessage = \"${mess}\"\npoints = \"${pts}\"\n[[check.pass]]\ntype= \"ServiceUpNot\"\narg1 = \"${arg1}\"\n"
+	;;
+
+
+"UserExists")
+	data=$(zenity --forms --text="Pass if User Exists On System" --add-entry "Message" --add-entry="Points" --add-entry="User" --separator=$'\034')
+	IFS=$'\034' read -r mess pts arg1 <<< "$data"
+	vulns="${vulns}\n[[check]]\nmessage = \"${mess}\"\npoints = \"${pts}\"\n[[check.pass]]\ntype= \"UserExists\"\narg1 = \"${arg1}\"\n"
+	;;
+
+"UserExistsNot")
+	data=$(zenity --forms --text="Pass if User does not exist On System" --add-entry "Message" --add-entry="Points" --add-entry="User" --separator=$'\034')
+	IFS=$'\034' read -r mess pts arg1 <<< "$data"
+	vulns="${vulns}\n[[check]]\nmessage = \"${mess}\"\npoints = \"${pts}\"\n[[check.pass]]\ntype= \"UserExistsNot\"\narg1 = \"${arg1}\"\n"
+	;;
+
+
+"FireWallUp")
+	data=$(zenity --forms --text="Pass if Firewall is Up, (Just Click Ok)" --separator=$'\034')
+	vulns="${vulns}\n[[check]]\nmessage = \"${mess}\"\npoints = \"${pts}\"\n[[check.pass]]\ntype= \"FireWallUp\"\n"
+	;;
+	
+"UserInGroup")
+	data=$(zenity --forms --text="Pass if User is in Group" --add-entry "Message" --add-entry="Points" --add-entry="User" --add-entry="Group" --separator=$'\034')
+	IFS=$'\034' read -r mess pts arg1 arg2 <<< "$data"
+	vulns="${vulns}\n[[check]]\nmessage = \"${mess}\"\npoints = \"${pts}\"\n[[check.pass]]\ntype= \"UserInGroup\"\narg1 = \"${arg1}\"\narg2 = \"${arg2}\"\n"
+	;;
+"UserInGroupNot")
+	data=$(zenity --forms --text="Pass if User is not in Group" --add-entry "Message" --add-entry="Points" --add-entry="User" --add-entry="Group" --separator=$'\034')
+	IFS=$'\034' read -r mess pts arg1 arg2 <<< "$data"
+	vulns="${vulns}\n[[check]]\nmessage = \"${mess}\"\npoints = \"${pts}\"\n[[check.pass]]\ntype= \"UserInGroupNot\"\narg1 = \"${arg1}\"\narg2 = \"${arg2}\"\n"
+	;;
+"GuestDisabledLDM")
+	data=$(zenity --forms --text="Pass if Gues is Disabled" --separator=$'\034')
+	vulns="${vulns}\n[[check]]\nmessage = \"${mess}\"\npoints = \"${pts}\"\n[[check.pass]]\ntype= \"GuestDisblaedLDM\"\n"
+	;;
+esac
+
+done
+
+FILE=/opt/aeacus/scoring.conf
+if [ ! -f "$FILE" ]; then
+    touch /opt/aeacus/scoring.conf
+fi
+
+echo "name = \"$name\"" >> /opt/aeacus/scoring.conf
+echo "title = \"$title\"" >> /opt/aeacus/scoring.conf
+echo "user = \"$user\"" >> /opt/aeacus/scoring.conf
+echo "os = \"$os\"" >> /opt/aeacus/scoring.conf
+echo "remote = \"$remote\"" >> /opt/aeacus/scoring.conf
+echo "local = \"$local\"" >> /opt/aeacus/scoring.conf
+echo "enddate = \"$end\"" >> /opt/aeacus/scoring.conf
+echo "nodestory = \"$destroy\"" >> /opt/aeacus/scoring.conf
+
+printf "$vulns" >> /opt/aeacus/scoring.conf
```

Comment:
Why is `printf` being used here?

Reply:
Echo Doesn't do the formatting correctly, for example \n.

Reply:
Use `echo -e`